generic3g/specs/is_in_set.h

```diff
@@ -0,0 +1,14 @@
+logical function FUNCNAME_(val) result(isin)
```

Comment:
Looks too much like trig.  I suggest:
```suggestion
logical function FUNCNAME_(val) result(is_in)
```

Reply:
Makes sense.